examples/spark/README.md

```diff
@@ -172,32 +172,44 @@ you should now see the workers in the UI as well. *Note:* The UI will have links
 to worker Web UIs. The worker UI links do not work (the links will attempt to
 connect to cluster IPs, which Kubernetes won't proxy automatically).
 
-## Step Three: Start your Spark driver to launch jobs on your Spark cluster
+## Step Three: Start the Zeppelin UI to launch jobs on your Spark cluster
 
-The Spark driver is used to launch jobs into Spark cluster. You can read more about it in
-[Spark architecture](https://spark.apache.org/docs/latest/cluster-overview.html).
+The Zeppelin UI pod can be used to launch jobs into the Spark cluster either via
+a web notebook frontend or the traditional Spark command line. Read more about
+[Zeppelin](https://zeppelin.incubator.apache.org/) and the
+[Spark architecture](https://spark.apache.org/docs/latest/cluster-overview.html)
+to understand more.
 
 ```console
-$ kubectl create -f examples/spark/spark-driver-controller.yaml
-replicationcontrollers/spark-driver-controller
+$ kubectl create -f examples/spark/zeppelin-controller.yaml
+replicationcontrollers/zeppelin-controller
 ```
 
-The Spark driver needs the Master service to be running.
+Zeppelin needs the Master service to be running.
 
-### Check to see if the driver is running
+### Check to see if zeppelin is running
 
 ```console
-$ kubectl get pods -lcomponent=spark-driver
-NAME                            READY     STATUS    RESTARTS   AGE
-spark-driver-controller-vwb9c   1/1       Running   0          1m
+$ kubectl get pods -lcomponent=zeppelin
+NAME                        READY     STATUS    RESTARTS   AGE
+zeppelin-controller-ja09s   1/1       Running   0          53s
 ```
 
+**Note:** The Zeppelin pod is large, so it may take a while to pull depending on
+  your network. The size of the Zeppelin pod is something we're working on, see
+  issue **TBD**.
```

Comment:
I was trying to figure out where to file this. I can file it in the main k/k repo, but to your later point, if we're going to break something out, it's actually an issue for a separate repo. I'll go ahead and file it here and we can copy it over.